catalog/message/schema/catalog-schema.json

```diff
@@ -1,39 +1,381 @@
+                    "items": {
+                        "$ref": "#/definitions/Multilanguage"
+                    }
+                },
+                "dct:issued": {
+                    "type": "object",
+                    "properties": {
+                        "@id": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "dct:modified": {
+                    "type": "object",
+                    "properties": {
+                        "@id": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "dct:title": {
+                    "type": "string"
+                },
+                "foaf:primaryTopic": {
+                    "$ref": "#/definitions/Resource"
+                }
+            }
+        },
+        "Resource": {
+            "type": "object",
+            "properties": {
+                "dcat:contractPoint": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dcat:keyword": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Multilanguage"
+                    }
+                },
+                "dcat:landingPage": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dcat:theme": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Reference"
+                    },
+                    "minItems": 1
+                },
+                "dct:conformsTo": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:creator": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:description": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Multilanguage"
+                    }
+                },
+                "dct:identifier": {
+                    "type": "string"
+                },
+                "dct:isReferencedBy": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:issued": {
+                    "$ref": "#/definitions/time"
+                },
+                "dct:language": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:license": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:modified": {
+                    "$ref": "#/definitions/time"
+                },
+                "dct:publisher": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:relation": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:title": {
```

Comment:
Exactly. `rdfs:Literal` is nasty to validate. Let's talk about it in our call today.

Reply:
Ok, as I more or less did not continue the discussion, let me trigger it again here.

> dct:title is a rdfs:Literal so it could be multilanguage

100% correct from the perspective of DCAT and RDF. 100% horrible from the perspective of the server logic ;-)
My implicit proposal by suggesting `type: string` is to reduce the degrees of freedom coming with rdfs:Literal and only allow one variant. 

While for `dct:description` - as I feel - multi language tags are plausible, for `dct:title` I somehow feel different. Not based on facts, though. I simply can imagine that users do not pay too much attention on the language of a title / use the same title across different languages, while they actually do care for e.g. `dct:description`...

What do you think? Would the possibilities of `dct:title` a multilanguage strings create more value than it would create overhead?

Reply:
This links to my comment in https://github.com/International-Data-Spaces-Association/ids-specification/issues/132#issuecomment-1631994313

I think this can be solved inside the context with type coercion, with perhaps a default language set to `"en"`.